test/DepthImageToLaserScanTest.cpp

```diff
@@ -36,6 +36,8 @@
 // Bring in gtest
 #include <gtest/gtest.h>
 
+#include <sensor_msgs/image_encodings.hpp>
```

Comment:
Thanks.  I moved my addition down (underneath the C system ones).  I did not rearrange the rest of the headers, in order to keep differences with upstream down.